content/posts/replacing-the-redisgraph-parser.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Lessons learned while replacing the RedisGraph parser"
+date: 2019-11-22
+authors:
+  - Jeffrey Lovitz
```

Comment:
authors:
  - author:
      name: "Jeffrey Lovitz"
      link: "https://github.com/jeffreylovitz"

Reply:
@gkorland This doesn't appear to be rendering properly, can I get an example?